src/renderer/components/chat/ChatListHelpers.tsx

```diff
@@ -24,17 +24,20 @@ export function debounceWithInit<ARGS extends Array<any>>(
   }
 }
 
-export function useMessageResults(queryStr: string | undefined) {
+export function useMessageResults(
+  queryStr: string | undefined,
+  chatId: number | null = null
+) {
   const [ids, setIds] = useState<number[]>([])
 
   const debouncedSearchMessages = useMemo(
     () =>
       debounceWithInit((queryStr: string | undefined) => {
-        DeltaBackend.call('messageList.searchMessages', queryStr || '', 0).then(
-          setIds
-        )
+        BackendRemote.rpc
+          .searchMessages(selectedAccountId(), queryStr || '', chatId)
```

Comment:
I dislike a bit that it's searchMessages(accountId, queryStr, chatId) instead of searchMessages(accountId, chatId, queryStr)

Reply:
sure that is a change we could revert, but should we? I mean I could argue that `queryStr` is more important and chatId can be null | number, in future maybe also | undefined, which means we could omit it here. https://github.com/Frando/yerpc/issues/10

Reply:
I think it's not so important if we have to change something with yerpc